src/v2/view-builder/views/DeviceChallengePollView.js

```diff
@@ -104,6 +122,27 @@ const Body = BaseForm.extend({
   },
 });
 
+const Footer = BaseFooter.extend({
+  links () {
+    let links = [];
+    const deviceChallenge = this.options.appState.get(
```

Comment:
Could be the same logic as line 36?

Reply:
I can override BaseView.initialize to make deviceChallenge available for both Body and Footer.

Reply:
oh, I see. then nvm for now.